content/blog/2016/2016-03-02-gsoc2016-announcement.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "Jenkins participates in Google Summer of Code 2016"
+tags:
+- general
+- jenkins2
+- gsoc
+author: oleg_nenashev
+---
+
+We are happy to announce that Jenkins project application has been accepted to [Google Summer of Code 2016](https://developers.google.com/open-source/gsoc/timeline). 
+Thanks to everybody who helped us to prepare the application and project ideas!
+
+This year we invite students to join the Jenkins community and to work together on the ongoing Jenkins 2.0 activities and other medium-term projects.
+
+In GSoC student projects we want to address the user experience in wide terms. 
+It includes not only user interface changes, but also availability improvements and major new features like [Pipeline-as-code](https://wiki.jenkins-ci.org/display/JENKINS/2.0+Pipeline+as+Code). 
+The activity involves all components of Jenkins: core, plugins, website and internal automation infrastructure.
+
+Our [project ideas](https://wiki.jenkins-ci.org/display/JENKINS/Google+Summer+Of+Code+2016) include:
+
+* Jenkins Web Interface Improvements 
+* Update Center 2.0
+* New generation of the Fingerprinting engine
+* External workspace manager
+* Integration of [Docker plugins](http://jenkins-ci.org/solutions/docker/) with Jenkins 2.0 features
+* Plugins for EDA and Embedded tools integration
+* Improvements of [Jenkins Support](https://wiki.jenkins-ci.org/display/JENKINS/Support+Core+Plugin) plugins 
+* Jenkins project infrastructure: website,  plugin documentation CD flows
+
+## If you are a student...
+
+1. Check out the project ideas [here](https://wiki.jenkins-ci.org/display/JENKINS/Google+Summer+Of+Code+2016).
+2. Select an interesting project idea or draft your own proposal.
+3. Introduce yourself the community and start the project discussion (see the guidelines [here](https://summerofcode.withgoogle.com/organizations/5668199471251456/)).
+4. Join us at GSoC office hours (time - TBA soon). We plan to have 2 meetings to cover timezones. The first meeting ETA - March 7th.
```

Comment:
I can handle the meetings on March 7th, so we can commit on it. BTW it would be useful to have mentors. Maybe we should just declare the data and cross our fingers